apps/dg/libraries/slickgrid/plugins/slick.rowselectionmodel.js

```diff
@@ -34,10 +34,12 @@
 
     function wrapHandler(handler) {
       return function () {
+        var result;
```

Comment:
I fixed it in both places for consistency in case the original were ever used or referenced down the road. If we don't anticipate using the original perhaps it would be clearer to remove it entirely.